### PR TITLE
Add the option to configure the popup timeout (or disable it)

### PR DESCRIPTION
**Pull request by tvbeek**
I added a new config value for the timeout. If it is less then 1 it will disable the timeout so it stays on the screen.

This will close https://github.com/themsaid/laravel-mail-preview/issues/24

### CODE REVIEW

**Comment by tvbeek** [comment]
@themsaid will you merge it? Else I will remove it

**Comment by akazorg** [comment]
This is a nice feature.

File: src/MailPreviewMiddleware.php [hunk 1]
```diff
@@ -56,13 +56,18 @@ private function addLinkToResponse($response, $previewPath)
...
 
-        $linkHTML .= "setTimeout(function(){ 
-        document.body.removeChild(document.getElementById('MailPreviewDriverBox')); 
-        }, 8000);";
+        if ($timeout > 0) {
+            $linkHTML .= "<script type=\"text/javascript\">";
+
+            $linkHTML .= "setTimeout(function(){
+            document.body.removeChild(document.getElementById('MailPreviewDriverBox'));
+            }, ". intval(config('mailpreview.popup_timeout', 8000)).");";
```
Comment (Jamesking56) [inline src/MailPreviewMiddleware.php]: Why not reuse `$timeout` instead of grabbing from config again?

Comment (tvbeek) [reply src/MailPreviewMiddleware.php]: Good point, I changed it in my last commit.

**Comment by akazorg** [comment]
Finally you found time for it! Thank you so much @themsaid. :)

**Pull request merged**